### PR TITLE
Complete all-providers GHA workflow.

### LINKED ISSUE #82: Ensure that latest build + deploy scripts are available in all providers
- [ ] Ensure that the major-version of each provider is reflected in this repo (@stack72 TO-DO)
- [ ] Swap out the use of MP to be a GHA
	- [ ] We build a dynamic matrix (https://tomasvotruba.com/blog/2020/11/16/how-to-make-dynamic-matrix-in-github-actions/) to fuel out GHA
	- [ ] generate scripts for ALL providers
	- [ ] foreach provider
		- [ ] checkout provider repo
		- [ ] copy the files across
		- [ ] open a PR

### PR DESCRIPTION
**Pull request by jkodroff**
Fixes #82

### CODE REVIEW

**Comment by jkodroff** [comment]
@stack72 Looks like it works!  https://github.com/pulumi/ci-mgmt/actions/runs/1495789851

File: .github/workflows/update-all-providers.yml [hunk 1]
```diff
@@ -1,20 +1,55 @@
...
+          cp -r ci-mgmt/provider-ci/providers/${{ matrix.provider }}/repo/. pulumi-${{ matrix.provider }}/.
+      - name: Create PR
+        uses: peter-evans/create-pull-request@v3
+        with:
+          path: pulumi-${{ matrix.provider }}
+          token: ${{ secrets.PULUMI_BOT_TOKEN }}
+          committer: Pulumi Bot <bot@pulumi.com>
+          author: Pulumi Bot <bot@pulumi.com>
+          commit-message: "Update GitHub Actions workflow files"
+          # TODO: Add the automation/merge label to automatically merge the PR if it passes all the tests.
```
Comment (stack72) [inline .github/workflows/update-all-providers.yml:48]: Love this!

File: .github/workflows/update-all-providers.yml [hunk 2]
```diff
@@ -1,20 +1,55 @@
 name: Update All Providers
 on:
   - workflow_dispatch
```
Comment (stack72) [inline .github/workflows/update-all-providers.yml:3]: I presume we are using a workflow_dispatch here so that we can control the rollout?

**Review by stack72 — APPROVED** [review]

File: .github/workflows/update-all-providers.yml (hunk 2 above)
Comment (jkodroff) [reply .github/workflows/update-all-providers.yml:3]: Correct.  I believe we only update all providers at particular times.

**Pull request merged**